aten/src/ATen/FunctionalTensorWrapper.cpp

```diff
@@ -23,6 +23,10 @@ void FunctionalTensorWrapper::set_constructor_metadata() {
   storage_ = functional_storage;
   storage_access_should_throw_ = false;
   key_set_ = c10::DispatchKeySet(c10::DispatchKey::Functionalize) | value_.key_set();
+  // All of the keys corresponding to functorch transforms should not be copied over.
+  // Functorch transforms all have their own wrapper tensors (e.g. BatchedTensorImpl) which expect
+  // to participate in the functorch transforms.
+  key_set_ = key_set_ & c10::functorch_transforms_ks;
```

Comment:
Should this be `key_set_ & ~c10::functorch_transforms_ks` or am I reading this wrong? Aren't we trying to remove the functorch keys from key_set ?

Reply:
yes 😛 thanks. I over-confidentally yolo'd CI last night without testing locally